corehq/motech/repeaters/models.py

```diff
@@ -945,12 +951,14 @@ def page(self, domain, skip, limit, repeater_id=None, state=None):
                 .prefetch_related('attempt_set'))
 
     def iter_partition(self, start, partition, total_partitions):
-        from django.db.models import F
+        from django.db.models import Exists, F, OuterRef
+        # look for is_paused=True since that subset is small
+        is_repeater_paused = Repeater.objects.filter(id=OuterRef("repeater_id"), is_paused=True)
```

Comment:
@gherceg 

Just curious to how does this work?
I though it was not possible to do all this in one query, so really excited to see this.